pipestat_example/data/frog1_data.txt

```diff
@@ -0,0 +1,4 @@
+ribbit
```

Comment:
there's no need to duplicate the data.

the data live independent of the pipeline and project, that was the whole point of this repository, to show how they are modules and can be independent.

Thus, the pipestat project example should not need to duplicate any data. it can simply use the same data the same way the other projects do.